esp/src/eclwatch/nls/pt-br/hpcc.js

```diff
@@ -353,18 +370,18 @@
     LogicalName: "Nome Lógico",
     Login: "Fazer um Login",
     Logout: "Fazer um Logout",
-    Logs: "Logs",
+    Logs: "Registros",
     LogVisualization: "Visualização de Log",
-    log_analysis_1: "log_analysis_1*",
-    LostFile2: "Arquivos perdidos",
+    LogVisualizationUnconfigured: "Visualização de Registro não está configurado, por favor, verifique as configurações do gerenciador de confguração",
     LostFile: "Arquivo perdido",
+    LostFile2: "Arquivos perdidos",
     LostFileMessage: "Um arquivo lógico que está faltando pelo menos uma parte de arquivo em ambos os locais primário e replicado em armazenamento. O arquivo lógico ainda é referenciado no servidor Dali. A exclusão do arquivo remove a referência do servidor Dali e de quaisquer partes restantes no disco.",
     Low: "Baixo",
     MachineInformation: "Informação da máquina",
     Machines: "Máquinas",
     Major: "Principal",
     ManagedBy: "Gerenciado Por",
-    ManagedByPlaceholder: "CN=HPCCAdmin,OU=users,OU=hpcc,DC=MyCo,DC=local",
+    ManagedByPlaceholder: "CN = HPCCAdmin, OU = usuários, OU = hpcc, DC = MyCo, DC = local",
```

Comment:
Need to remove all the spaces. Didn't see that.
CN=HPCCAdmin,OU=usuários,OU=hpcc,DC=MyCo,DC=local